service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomApplicationLoader.scala

```diff
@@ -305,7 +288,7 @@ private[server] object ActorSystemProvider {
   @deprecated(message = "prefer method using typesafe Config instead", since = "1.4.0")
   def start(configuration: Configuration, environment: Environment,
             serializerRegistry: Option[JsonSerializerRegistry]): (ActorSystem, () => Future[Unit]) = {
-    start(configuration.underlying, environment, serializerRegistry)
+    start(configuration, environment, serializerRegistry)
```

Comment:
This is now just calling itself in an infinite recursion, but this method can be completely removed, since it's in a private object and is no longer used.